library/billrep.inc

```diff
@@ -171,9 +171,9 @@ function billCodesList($list, $skip = "()")
     }
 
     if ($skip == "()") {
-        sqlStatement("update billing set billed=1 where id in $list");
+        sqlStatement("UPDATE billing SET billed=1 WHERE id IN ?", array($list));
```

Comment:
TODO - look up best way to bind IN statements

Reply:
yep, this won't work since the $list will look something like this:
('4','887','4545')

Rec reverting this and we can work on how to escape this in a separate project (will involve reworking where $list is built out and instead building an array; then will be easier to integrate escaping into the query).